src/Datadog.Trace/Tagging/HttpTags.cs

```diff
@@ -0,0 +1,42 @@
+using Datadog.Trace.ExtensionMethods;
+
+namespace Datadog.Trace.Tagging
+{
+    internal class HttpTags : CommonTags
+    {
+        internal static readonly IProperty<string>[] HttpTagsProperties =
+            CommonTagsProperties.Concat(
+                new Property<HttpTags, string>(Trace.Tags.Env, t => t.Environment, (t, v) => t.Environment = v),
+                new Property<HttpTags, string>(Trace.Tags.Version, t => t.Version, (t, v) => t.Version = v),
```

Comment:
`CommonTagsProperties`  doesn't have both already?

Reply:
You're right, gonna fix that

Reply:
I don't want to use reflection at runtime for performance concerns, but I also realize it's too dangerous as-is. Gonna add a unit test to dynamically inspect tags and ensure they're properly declared